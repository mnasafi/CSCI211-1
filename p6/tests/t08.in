point 22 0 \
point 33 0 /
point 34 0 ^
point 35 0 \
point 24 1 )
point 32 1 /
point 33 1 /
point 34 1 ^
point 35 1 \
point 36 1 \
point 21 2 (
point 31 2 /
point 32 2 /
point 36 2 \
point 37 2 \
point 23 3 )
point 30 3 /
point 31 3 /
point 37 3 \
point 38 3 \
point 22 4 _
point 23 4 _
point 29 4 /
point 30 4 /
point 38 4 \
point 39 4 \
point 21 5 |
point 22 5 =
point 23 5 ^
point 24 5 |
point 28 5 /
point 29 5 /
point 34 5 _
point 39 5 \
point 40 5 \
point 19 6 _
point 20 6 _
point 21 6 |
point 22 6 =
point 24 6 |
point 25 6 _
point 26 6 _
point 27 6 /
point 28 6 /
point 33 6 (
point 34 6 +
point 35 6 )
point 40 6 \
point 41 6 \
point 18 7 /
point 19 7 L
point 20 7 L
point 21 7 L
point 22 7 L
point 23 7 L
point 24 7 L
point 25 7 L
point 26 7 /
point 27 7 /
point 34 7 ~
point 41 7 \
point 42 7 \
point 17 8 /
point 18 8 L
point 19 8 L
point 20 8 L
point 21 8 L
point 22 8 L
point 23 8 L
point 24 8 L
point 25 8 /
point 26 8 /
point 42 8 \
point 43 8 \
point 16 9 /
point 17 9 L
point 18 9 L
point 19 9 L
point 20 9 L
point 21 9 L
point 22 9 L
point 23 9 L
point 24 9 /
point 25 9 /
point 43 9 \
point 44 9 \
point 15 10 /
point 16 10 L
point 17 10 L
point 18 10 L
point 19 10 L
point 20 10 L
point 21 10 L
point 22 10 L
point 23 10 /
point 24 10 /
point 27 10 |
point 28 10 ~
point 29 10 [
point 30 10 |
point 31 10 ]
point 32 10 ~
point 33 10 |
point 35 10 |
point 36 10 ~
point 37 10 [
point 38 10 |
point 39 10 ]
point 40 10 ~
point 41 10 |
point 44 10 \
point 45 10 \
point 15 11 ^
point 16 11 |
point 18 11 [
point 19 11 |
point 20 11 ]
point 22 11 /
point 23 11 /
point 27 11 |
point 29 11 [
point 30 11 |
point 31 11 ]
point 33 11 |
point 35 11 |
point 37 11 [
point 38 11 |
point 39 11 ]
point 41 11 |
point 45 11 \
point 46 11 \
point 16 12 |
point 18 12 [
point 19 12 |
point 20 12 ]
point 22 12 ^
point 23 12 |
point 27 12 |
point 28 12 _
point 29 12 [
point 30 12 |
point 31 12 ]
point 32 12 _
point 33 12 |
point 35 12 |
point 36 12 _
point 37 12 [
point 38 12 |
point 39 12 ]
point 40 12 _
point 41 12 |
point 45 12 |
point 46 12 ^
point 13 13 _
point 14 13 _
point 15 13 _
point 16 13 |
point 17 13 _
point 18 13 _
point 19 13 _
point 20 13 _
point 21 13 _
point 22 13 _
point 23 13 |
point 45 13 |
point 12 14 /
point 13 14 L
point 14 14 L
point 15 14 L
point 16 14 L
point 17 14 L
point 18 14 L
point 19 14 L
point 20 14 L
point 21 14 L
point 22 14 L
point 23 14 |
point 24 14 _
point 25 14 _
point 26 14 _
point 27 14 _
point 28 14 _
point 29 14 _
point 30 14 _
point 31 14 _
point 32 14 _
point 33 14 _
point 34 14 _
point 35 14 _
point 36 14 _
point 37 14 _
point 38 14 _
point 39 14 _
point 40 14 _
point 41 14 _
point 42 14 _
point 43 14 _
point 44 14 _
point 45 14 |
point 10 15 /
point 11 15 L
point 12 15 L
point 13 15 L
point 14 15 L
point 15 15 L
point 16 15 L
point 17 15 L
point 18 15 L
point 19 15 L
point 20 15 L
point 21 15 L
point 22 15 /
point 23 15 L
point 24 15 L
point 25 15 L
point 26 15 L
point 27 15 L
point 28 15 L
point 29 15 L
point 30 15 L
point 31 15 L
point 32 15 L
point 33 15 L
point 34 15 L
point 35 15 L
point 36 15 L
point 37 15 L
point 38 15 L
point 39 15 L
point 40 15 L
point 41 15 L
point 42 15 L
point 43 15 L
point 44 15 L
point 45 15 L
point 46 15 L
point 47 15 \
point 10 16 ^
point 11 16 |
point 12 16 |
point 13 16 ^
point 14 16 ^
point 15 16 ^
point 16 16 ^
point 17 16 ^
point 18 16 ^
point 19 16 ^
point 20 16 ^
point 21 16 /
point 22 16 L
point 23 16 L
point 24 16 L
point 25 16 L
point 26 16 L
point 27 16 L
point 28 16 L
point 29 16 L
point 30 16 L
point 31 16 L
point 32 16 L
point 33 16 L
point 34 16 L
point 35 16 L
point 36 16 L
point 37 16 L
point 38 16 L
point 39 16 L
point 40 16 L
point 41 16 L
point 42 16 L
point 43 16 L
point 44 16 L
point 45 16 L
point 46 16 L
point 47 16 L
point 48 16 \
point 11 17 |
point 12 17 |
point 14 17 |
point 15 17 ~
point 16 17 [
point 17 17 |
point 18 17 ]
point 19 17 ~
point 20 17 |
point 21 17 ^
point 22 17 ^
point 23 17 |
point 24 17 |
point 25 17 ^
point 26 17 ^
point 27 17 ^
point 28 17 ^
point 29 17 ^
point 30 17 ^
point 31 17 ^
point 32 17 ^
point 33 17 ^
point 34 17 ^
point 35 17 |
point 36 17 |
point 37 17 ^
point 38 17 |
point 39 17 ~
point 40 17 [
point 41 17 |
point 42 17 ]
point 43 17 ~
point 44 17 |
point 45 17 ^
point 46 17 |
point 47 17 |
point 48 17 ^
point 49 17 ^
point 11 18 |
point 12 18 |
point 14 18 |
point 16 18 [
point 17 18 |
point 18 18 ]
point 20 18 |
point 23 18 |
point 24 18 |
point 27 18 |
point 28 18 ~
point 29 18 ~
point 30 18 ~
point 31 18 ~
point 32 18 |
point 35 18 |
point 36 18 |
point 38 18 |
point 40 18 [
point 41 18 |
point 42 18 ]
point 44 18 |
point 46 18 |
point 47 18 |
point 11 19 |
point 12 19 |
point 14 19 |
point 15 19 _
point 16 19 [
point 17 19 |
point 18 19 ]
point 19 19 _
point 20 19 |
point 23 19 |
point 24 19 |
point 27 19 |
point 29 19 [
point 30 19 ]
point 32 19 |
point 35 19 |
point 36 19 |
point 38 19 |
point 39 19 _
point 40 19 [
point 41 19 |
point 42 19 ]
point 43 19 _
point 44 19 |
point 46 19 |
point 47 19 |
point 11 20 |
point 12 20 |
point 13 20 _
point 14 20 _
point 15 20 _
point 16 20 _
point 17 20 _
point 18 20 _
point 19 20 _
point 20 20 _
point 21 20 _
point 22 20 _
point 23 20 |
point 24 20 |
point 27 20 |
point 31 20 o
point 32 20 |
point 35 20 |
point 36 20 |
point 37 20 _
point 38 20 _
point 39 20 _
point 40 20 _
point 41 20 _
point 42 20 _
point 43 20 _
point 44 20 _
point 45 20 _
point 46 20 |
point 47 20 |
point 9 21 .
point 10 21 '
point 11 21 |
point 12 21 |
point 13 21 ]
point 14 21 [
point 15 21 ]
point 16 21 [
point 17 21 ]
point 18 21 [
point 19 21 ]
point 20 21 [
point 21 21 ]
point 22 21 [
point 23 21 |
point 24 21 |
point 27 21 |
point 29 21 [
point 30 21 ]
point 32 21 |
point 35 21 |
point 36 21 |
point 37 21 [
point 38 21 ]
point 39 21 [
point 40 21 ]
point 41 21 [
point 42 21 ]
point 43 21 [
point 44 21 ]
point 45 21 [
point 46 21 |
point 47 21 |
point 48 21 .
point 49 21 '
point 50 21 .
point 8 22 .
point 9 22 "
point 10 22 '
point 11 22 |
point 12 22 |
point 13 22 [
point 14 22 ]
point 15 22 [
point 16 22 ]
point 17 22 [
point 18 22 ]
point 19 22 [
point 20 22 ]
point 21 22 [
point 22 22 ]
point 23 22 |
point 24 22 |
point 25 22 _
point 26 22 -
point 27 22 `
point 28 22 -
point 29 22 -
point 30 22 -
point 31 22 -
point 32 22 '
point 33 22 -
point 34 22 _
point 35 22 |
point 36 22 |
point 37 22 ]
point 38 22 [
point 39 22 ]
point 40 22 [
point 41 22 ]
point 42 22 [
point 43 22 ]
point 44 22 [
point 45 22 ]
point 46 22 |
point 47 22 |
point 48 22 "
point 49 22 .
point 50 22 "
point 0 23 s
point 1 23 o
point 2 23 u
point 3 23 r
point 4 23 c
point 5 23 e
point 6 23 :
point 8 23 w
point 9 23 w
point 10 23 w
point 11 23 .
point 12 23 a
point 13 23 l
point 14 23 b
point 15 23 i
point 16 23 n
point 17 23 o
point 18 23 b
point 19 23 l
point 20 23 a
point 21 23 c
point 22 23 k
point 23 23 s
point 24 23 h
point 25 23 e
point 26 23 e
point 27 23 p
point 28 23 .
point 29 23 c
point 30 23 o
point 31 23 m
point 32 23 /
point 33 23 t
point 34 23 x
point 35 23 t
point 36 23 /
point 37 23 a
point 38 23 s
point 39 23 c
point 40 23 i
point 41 23 i
point 42 23 .
point 43 23 p
point 44 23 h
point 45 23 p
