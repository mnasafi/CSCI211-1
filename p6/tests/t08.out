                      \          /^\                        
                        )       //^\\                       
                     (         //   \\                      
                       )      //     \\                     
                      __     //       \\                    
                     |=^|   //    _    \\                   
                   __|= |__//    (+)    \\                  
                  /LLLLLLL//      ~      \\                 
                 /LLLLLLL//               \\                
                /LLLLLLL//                 \\               
               /LLLLLLL//  |~[|]~| |~[|]~|  \\              
               ^| [|] //   | [|] | | [|] |   \\             
                | [|] ^|   |_[|]_| |_[|]_|   |^             
             ___|______|                     |              
            /LLLLLLLLLL|_____________________|              
          /LLLLLLLLLLL/LLLLLLLLLLLLLLLLLLLLLLLL\            
          ^||^^^^^^^^/LLLLLLLLLLLLLLLLLLLLLLLLLL\           
           || |~[|]~|^^||^^^^^^^^^^||^|~[|]~|^||^^          
           || | [|] |  ||  |~~~~|  || | [|] | ||            
           || |_[|]_|  ||  | [] |  || |_[|]_| ||            
           ||__________||  |   o|  ||_________||            
         .'||][][][][][||  | [] |  ||[][][][][||.'.         
        ."'||[][][][][]||_-`----'-_||][][][][]||"."         
source: www.albinoblacksheep.com/txt/ascii.php              
